CoverMyDotNet.Samples/Program.cs

```diff
@@ -52,21 +64,166 @@ public static void MultipleRequests(Client client)
 					FirstName = "Tom",
 					LastName = "test2",
 					DateOfBirth = DateTime.Parse("4/4/1972")
+				},
+				Payer = new PayerAttributes()
+				{
+					Bin = "773836",
+					PCN = "MOCKPBM",
+					GroupId = "ABC1",
```

Comment:
The data for these Payers is repeated.  Is it worthwhile to DRY these up into a single instance that gets used for each, or is this intentional so that the code is more explicit and/or can easily be copy-pasted by our consumers?